spring-boot-project/spring-boot-starters/README.adoc

```diff
@@ -35,7 +35,7 @@ do as they were designed before this was clarified.
 | https://github.com/amqphub/amqp-10-jms-spring-boot
 
 | https://rocketmq.apache.org/[Apache RocketMQ]
-| https://github.com/ThierrySquirrel/rocketmq-spring-boot-starter
+| https://github.com/rocketmq/rocketmq-spring-boot-starter
```

Comment:
The old starter is 404. https://github.com/rocketmq/rocketmq-spring-boot-starter hasn't been updated for 4 years. I think we should remove this one.

Reply:
Done.